Control.c

```diff
@@ -22,6 +26,12 @@ int main(void)
 	// Set default status to None
 	Control_GlobalStatus = None;
 	
+	// Set default state to Neither
+	Control_GlobalState = Undefined;
+	
+	// Set default bool to false
+	isEmpty = false;
```

Comment:
I was going to use this as a check. But never did. I might still need it, not sure yet since it's not working yet.